### DISCUSSION TITLE
如果出一个手把手教你做开源的课程，你想听到哪些内容呢

### DISCUSSION THREAD

**Discussion by shengxinjing** [tutorial-kit/community#6] [Ideas]
1. 玩转github使用   （Git和开源基本礼仪）
2. 开源博客实战        
3. 好用的第三方那个库
    * eslint
    * pnpm
    * typescript
    * .....
5. 开源项目实战
    * 命令行工具
    * Vue 组件
    * React 组件
    * Vite插件
    * VScode插件
    * Chrome插件
    * 参与热门开源项目
   

其他欢迎讨论

**Comment by lilei2603** [comment]
玩转github使用和Vite插件

**Reply by shengxinjing** [reply]
这两个都在计划中 😄

**Comment by ErKeLost** [comment]
Vite插件  参与热门开源项目

**Reply by shengxinjing** [reply]
get

**Comment by bbcvc** [comment]
用vue3做个聊天软件   类似discord那种🤩

**Reply by shengxinjing** [reply]
这个有一丢丢超纲  这个得带后端

**Comment by 3000-codes** [comment]
AST与babel

**Comment by higuaifan** [comment]
git action和 git projects的应用

**Comment by youuss** [comment]
针对不同的类型项目如何选型、如何设计
